gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/modules/restli/FlowConfigsV2ResourceHandler.java

```diff
@@ -60,14 +63,22 @@
 import org.apache.gobblin.runtime.api.SpecNotFoundException;
 import org.apache.gobblin.runtime.spec_catalog.AddSpecResponse;
 import org.apache.gobblin.runtime.spec_catalog.FlowCatalog;
+import org.apache.gobblin.runtime.util.InjectionNames;
+import org.apache.gobblin.service.FlowConfig;
+import org.apache.gobblin.service.FlowConfigLoggedException;
+import org.apache.gobblin.service.FlowId;
+import org.apache.gobblin.service.FlowStatusId;
+import org.apache.gobblin.service.RequesterService;
+import org.apache.gobblin.service.Schedule;
+import org.apache.gobblin.service.ServiceConfigKeys;
 import org.apache.gobblin.util.ConfigUtils;
 
 
-/**
- * A {@link FlowConfigsResourceHandler} that handles Restli locally.
- */
 @Slf4j
-public class FlowConfigResourceLocalHandler implements FlowConfigsResourceHandler {
+public class FlowConfigsV2ResourceHandler {
```

Comment:
why not implement a `FlowConfigsV2ResourceHandlerInterface`, like flowExecutions does?

Reply:
```// Unlike FlowConfigsV2ResourceHandler, FlowExecutionResourceHandlerInterface is an interface rather than a class because it's implementation needs
// classes from gobblin-service module, and adding gobblin-service as a dependency will cause circular dependency```

now i am thinking to move FlowExecutionResource that needs it to gobblin-service module, so I can get rid of FlowExecutionResourceHandlerInterface. wdyt?